kdump/include/runtest.sh

```diff
@@ -339,21 +339,45 @@ ClearReport()
 
 #  Common Kdump/Crash Functions
 
+# Install/Upgrade Kexec-tools and related packages
 PrepareKdump()
 {
-    if [ ! -f "${K_REBOOT}" ]; then
-        rpm -q kexec-tools || {
-            # On Fedora, kexec-tools is not installed by default.
-            # install kexec-tools ans enabled kdump service.
-            InstallPackages kexec-tools
-            systemctl enable kdump.service || chkconfig kdump on
-        }
+    rpm -q kexec-tools || {
+        # On Fedora, kexec-tools is not installed by default.
+        # Install kexec-tools and enable kdump service.
+        InstallPackages kexec-tools
+        systemctl enable kdump.service || chkconfig kdump on
+
+        # Back up configurations if kexec-tools is installed for the first time
+        [ -f $"${KDUMP_CONFIG}" -a ! -f "${KDUMP_CONFIG}.bk" ] && cp "${KDUMP_CONFIG}" "${KDUMP_CONFIG}.bk"
+        [ -f $"${KDUMP_SYS_CONFIG}" -a ! -f "${KDUMP_SYS_CONFIG}.bk" ] && cp "${KDUMP_SYS_CONFIG}" "${KDUMP_SYS_CONFIG}.bk"
```

Comment:
Oh, haha, good thing I asked, then! I thought it was some obscure syntax thing I didn't know and I wanted to learn it 😂

You forgot to fix these two or...?

Reply:
And can you update the branch while at it? Just realised it's out-of-date.

Reply:
Thanks...Hold on, still updating the code. Got some issues on the DNS..
Fingers crossed that there will be no more ${$var} error.